doc/superbuild/source/developers/testing/CI.rst

```diff
@@ -123,125 +128,103 @@ the jobs in a stage succeed, the pipeline moves on to the next stage. If any job
-
-* The ``variables:`` keyword defines environment variables shared by all stages
-  and jobs. Variables can be used to alter the CI pipeline behavior when
-  launching a job from the GitLab "Run pipeline" UI or from the GitLab pipeline
-  scheduling UI. See the ``.gitlab-ci.yml`` file for details about all of the
-  available variables and what they do.
-
-  .. code-block:: YAML
-
-     variables:
-       GIT_SUBMODULE_STRATEGY: recursive
-       ALLOC_NAME: ${CI_PROJECT_NAME}_ci_${CI_PIPELINE_ID}
-       BUILD_ROOT: ${CI_PROJECT_DIR}
-       # ...
-
-* The ``stages:`` keyword defines independent CI stages targeting a specific
-  test machine following the prefix naming convention:
-
-  * ``q_`` jobs run on Quartz
-  * ``l_`` jobs run on Lassen
-
-  .. code-block:: YAML
-
-     stages:
-       - q_build_and_test
-       - l_build_and_test
-       # ...
-
-* Several hidden job templates (job names start with ``.``) are defined for
-  specific architectures and batch queue systems. These jobs provide the batch
-  system command to run the ``build_and_test.sh`` script that configures,
-  builds, and tests SUNDIALS.
-
-  .. code-block:: YAML
-
-     .build_toss_3_x86_64_ib_script:
-       script:
-       - echo ${ALLOC_NAME}
-       - srun -p pdebug -N 1 -n ${NCPUS} --interactive -t ${DEFAULT_TIME}
-         --job-name=${ALLOC_NAME} .gitlab/build_and_test.sh
-
-     # ...
-
-* The ``include:`` keyword loads YAML files defining the jobs for specific
-  machines.
-
-  .. code-block:: YAML
-
-     include:
-       - local: .gitlab/quartz-jobs.yml
-       - local: .gitlab/lassen-jobs.yml
-       # ...
-
-
-CI Jobs
-^^^^^^^
-
-As noted above, each stage in the CI pipeline corresponds to testing on a
-specific machine. For example, jobs run on Lassen are associated with the
-``l_build_and_test`` stage. The actual jobs to run are defined in the YAML
-file ``.gitlab/lassen-jobs.yml``.
-
-The Lassen build and test jobs inherit from three job templates:
-
-* ``.build_blueos_3_ppc64le_ib_script`` executes the LSF command to run the
-  testing script.
+.. code-block:: YAML
 
-* ``.on_lassen`` defines the tags (``tags:`` keyword) to select a shell runner
-  on Lassen and the rules (``rules:`` keyword) for when a job should run.
+  tioga_rocmcc_620_tpls:
+    parallel:
+      matrix:
+        - COMPILER_SPEC: rocmcc@6.2.0
+          AMDGPU_TARGET: [gfx90a]
+    variables:
+      # have to use ginkgo@master because our spack version does not have ginkgo@1.8.0: yet (which seems to be needed)
+      # similarly, we need a newer magma than available to compile with 'rocm@6:' so we turn it off
+      SPEC: "%${COMPILER_SPEC} cstd=99 cxxstd=14 precision=double amdgpu_target=${AMDGPU_TARGET} +rocm+mpi~magma+ginkgo+kokkos ^ginkgo@master+rocm amdgpu_target=${AMDGPU_TARGET} ^kokkos+rocm amdgpu_target=${AMDGPU_TARGET}"
+    before_script:
+      - module load rocmcc/6.2.0-magic
+    extends: [.sundials_job_on_tioga]
```

Comment:
Indent three space to align with `code-block::`
```suggestion
   tioga_rocmcc_620_tpls:
     parallel:
       matrix:
         - COMPILER_SPEC: rocmcc@6.2.0
           AMDGPU_TARGET: [gfx90a]
     variables:
       # have to use ginkgo@master because our spack version does not have ginkgo@1.8.0: yet (which seems to be needed)
       # similarly, we need a newer magma than available to compile with 'rocm@6:' so we turn it off
       SPEC: "%${COMPILER_SPEC} cstd=99 cxxstd=14 precision=double amdgpu_target=${AMDGPU_TARGET} +rocm+mpi~magma+ginkgo+kokkos ^ginkgo@master+rocm amdgpu_target=${AMDGPU_TARGET} ^kokkos+rocm amdgpu_target=${AMDGPU_TARGET}"
     before_script:
       - module load rocmcc/6.2.0-magic
     extends: [.sundials_job_on_tioga]
```